plugin/src/main/java/net/mcjukebox/plugin/bukkit/commands/RegionCommand.java

```diff
@@ -25,7 +26,17 @@ public RegionCommand(RegionManager regionManager) {
     public boolean execute(CommandSender dispatcher, String[] args) {
         // region add <id> <url>
         if (args.length == 3 && args[0].equalsIgnoreCase("add")){
-            MCJukebox.getInstance().getRegionManager().addRegion(args[1], args[2]);
+            String url = args[2];
+            
+            if(!UrlUtils.isValidURI(url)) {
+                MessageUtils.sendMessage(dispatcher, "command.invalidUrl");
```

Comment:
Do you think we can just return `false` here? That should lead to the command syntax being printed and I think that's enough to indicate that we didn't get the argument we expected.

Reply:
I think returning the message and then `true` would make it more clear to users. Returning false results in the same list as /jukebox help.
This error tells the user what exactly they did wrong. Less technical users might enter an URL like domain.com/file.mp3, omitting the http(s)://, which I believe prevents the file from playing. To a user the given URL might be correct, but it is not, which results in confusion.

Reply:
That makes sense :)